The process tests run /bin/cat, /bin/false, /bin/pwd and /usr/bin/env, which a
build sandbox does not have. git_process looks a bare name up on PATH.

--- a/tests/util/process/start.c
+++ b/tests/util/process/start.c
@@ -44,7 +44,7 @@
       defined(__MidnightBSD__) || defined(__DragonFly__)
 	const char *args_array[] = { "/usr/bin/false" };
 #else
-	const char *args_array[] = { "/bin/false" };
+	const char *args_array[] = { "false" };
 #endif
 
 	git_process *process;
@@ -155,7 +155,7 @@
 #ifdef GIT_WIN32
 	const char *args_array[] = { "C:\\Windows\\System32\\cmd.exe", "/c", cat_cmd.ptr };
 #else
-	const char *args_array[] = { "/bin/cat" };
+	const char *args_array[] = { "cat" };
 #endif
 
 	git_process *process;
@@ -188,7 +188,7 @@
 /*
 void test_process_start__catch_sigterm(void)
 {
-	const char *args_array[] = { "/bin/cat" };
+	const char *args_array[] = { "cat" };
 
 	git_process *process;
 	git_process_options opts = GIT_PROCESS_OPTIONS_INIT;
@@ -241,7 +241,7 @@
 	const char *args_array[] = { "C:\\Windows\\System32\\cmd.exe", "/c", pwd_cmd.ptr };
 	char *startwd = "C:\\";
 #else
-	const char *args_array[] = { "/bin/pwd" };
+	const char *args_array[] = { "pwd" };
 	char *startwd = "/";
 #endif
 
@@ -277,7 +277,7 @@
 	const char *args_array[] = { "C:\\Windows\\System32\\cmd.exe", "/c", pwd_cmd.ptr };
 	char *startwd = "C:\\a\\b\\z\\y\\not_found";
 #else
-	const char *args_array[] = { "/bin/pwd" };
+	const char *args_array[] = { "pwd" };
 	char *startwd = "/a/b/z/y/not_found";
 #endif
 
--- a/tests/util/process/env.c
+++ b/tests/util/process/env.c
@@ -11,7 +11,7 @@
 #ifdef GIT_WIN32
 	git_str_printf(&env_cmd, "%s/env.cmd", cl_fixture("process"));
 #else
-	git_str_puts(&env_cmd, "/usr/bin/env");
+	git_str_puts(&env_cmd, "env");
 #endif
 
 	cl_git_pass(git_vector_init(&env_result, 32, git__strcmp_cb));
